src/nspawn/nspawn.c

```diff
@@ -1027,6 +1054,11 @@ static int parse_argv(int argc, char *argv[]) {
                 return -EINVAL;
         }
 
+        if (arg_network_bridge && arg_network_zone) {
+                log_error("--network-bridge= and --network-zone= may not be combined.");
+                return -EINVAL;
+        }
```

Comment:
Well. both control the "primary" interface of the container, i.e. the one that is called "host0" in the container and vb-xyz on the host. Since a specific iface can only be part of one bridge at a time it hence makes no sense to allow both in parallel or  either multiple times.

if people want multiple links between the host and the container they should use --network-veth-extra= which allows you to configure any number of links you like but isnt that automatic then as the generic .network files we ship dont apply.

I think have a simple automatic scheme that only covers a single link per container on one hand plus a ore minimal, more manual way to define arbitrary additional links that require additional config in networkd but allow you to do anything you like on the other is a good combination.